pygmt/alias.py

```diff
@@ -0,0 +1,124 @@
+"""
+The PyGMT alias system to convert PyGMT long-form arguments to GMT's short-form.
+"""
+
+from collections.abc import Mapping, Sequence
+from typing import Any, Literal
+
+from pygmt.exceptions import GMTValueError
+from pygmt.helpers.utils import is_nonstr_iter, sequence_join
+
+
+def _to_string(
+    value: Any,
+    prefix: str = "",  # Default to an empty string to simplify the code logic.
+    mapping: Mapping | None = None,
+    separator: Literal["/", ","] | None = None,
+    size: int | Sequence[int] | None = None,
+    ndim: int = 1,
+    name: str | None = None,
```

Comment:
Missing docstring for these three parameters?

Reply:
Added in 232ff52e291a03426be72b86d42ab3cd56b19e35.